libpod/runtime.go

```diff
@@ -0,0 +1,76 @@
+package libpod
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/libpod/ctr"
+	"github.com/kubernetes-incubator/cri-o/libpod/pod"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+var (
+	// ErrNotImplemented indicates that functionality is not yet implemented
+	ErrNotImplemented = fmt.Errorf("NOT IMPLEMENTED")
+)
+
+// Runtime is the core libpod runtime
+type Runtime struct {
```

Comment:
By storage server, do you mean the paths where we store container information? I think that's important to expose to API consumers.